examples/pytorch/gcn/README.md

```diff
@@ -2,33 +2,29 @@ Graph Convolutional Networks (GCN)
 ============
 
 - Paper link: [https://arxiv.org/abs/1609.02907](https://arxiv.org/abs/1609.02907)
-- Author's code repo: [https://github.com/tkipf/gcn](https://github.com/tkipf/gcn). Note that the original code is 
-implemented with Tensorflow for the paper. 
+- Author's code repo: [https://github.com/tkipf/gcn](https://github.com/tkipf/gcn). Note that the original implementation is based on Tensorflow.
 
-Dependencies
-------------
-- PyTorch 0.4.1+
-- requests
-
-``bash
-pip install torch requests
-``
-
-Codes
+Available examples
 -----
-The folder contains three implementations of GCN:
-- `gcn.py` uses DGL's predefined graph convolution module.
-- `gcn_mp.py` uses user-defined message and reduce functions.
-Modify `train.py` to switch between different implementations.
+The folder includes an implementations of GCN using DGL intrinsic graph convolution module.
 
-Results
+How to run
 -------
 
-Run with following (available dataset: "cora", "citeseer", "pubmed")
+### DGL intrinsic GCN module
+
+Run with the following (available dataset: "cora", "citeseer", "pubmed")
 ```bash
-python3 train.py --dataset cora --gpu 0 --self-loop
+python3 train.py --dataset cora
+```
+Results:
+```
+Test Accuracy: 0.8140
 ```
 
+Summary
+-------
 * cora: ~0.810 (0.79-0.83) (paper: 0.815)
 * citeseer: 0.707 (paper: 0.703)
 * pubmed: 0.792 (paper: 0.790)
+
```

Comment:
Have you re-performed the experiments?

Reply:
Yes, I did it again for this version, and let me list the results here for reference:
cora: 0.809
citeseer: 0.714
pubmed: 0.801